pages/api/activities/index.js

```diff
@@ -7,6 +7,19 @@ export default async function handler(request, response) {
   if (request.method === "GET") {
     const activities = await Activity.find();
     return response.status(200).json(activities);
+  }
+
+  if (request.method === "POST") {
+    try {
+      const activityData = request.body;
+      console.log("Received activity data:", activityData); // Konsolenausgabe der empfangenen Request-Daten
```

Comment:
```suggestion
```
Always delete console.logs including any helpers or data before pushing. This can potentially cause insecurities in your website (if that happend with userdata for example :)